.golangci.yml

```diff
@@ -9,5 +9,6 @@ run:
 linters:
   enable:
     - revive
+    - staticcheck
```

Comment:
Minor:  It's non-obvious from this config file, but there is an important note in the upstream golangci-lint readme:  

    It's not the same thing as the staticcheck binary. The author of staticcheck 
    doesn't support or approve the use of staticcheck as a library inside golangci-
    lint.

Maybe consider if it would help future maintainers to add a comment like:

`# N/B: Not the same as https://staticcheck.dev/`

Though, oddly enough, clicking on 'staticcheck' on the golangci-lint page, brings you to that site.  So I'm not 100% on what to think about the readme statement :confused:

Reply:
Ah, nice, that probably explains why CI is passing. Thanks,